vtex-apps/product-summary
# Fix ProductImage crashing when there is no image to render

#### What is the purpose of this pull request?
As the title says

#### How should this be manually tested?
1. [workspace](https://klynger--storecomponents.myvtex.com/home---decor/?__disableSSR)
2. To test this I removed all images in code to reproduce the use case, but if you want to see a product with an image, just open the quick view of the second item and it will magically display one

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/8517023/81826429-26210380-950e-11ea-91bb-6fc51d025e6d.png)



#### Types of changes
